src/main/java/com/wolox/training/model/Book.java

```diff
@@ -0,0 +1,144 @@
+package com.wolox.training.model;
+
+import static java.util.Objects.requireNonNull;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+public final class Book {
+
+    private @Id
+    Long idBook;
```

Comment:
Add`@GeneratedValue(strategy = GenerationType.SEQUENCE)`, to automatically generate the id value.
by convection we define the attributes with the following order
`@Id`
`@GeneratedValue(strategy = GenerationType.SEQUENCE)`
`private Long idBook`